partition/bootloader.go

```diff
@@ -146,3 +155,13 @@ func MarkBootSuccessful(bootloader Bootloader) error {
 
 	return bootloader.SetBootVars(m)
 }
+
+func reboot() error {
```

Comment:
Also I think we should give this a better name. like regularSystemReboot() so it name expresses a bit more that this is the std. way of doing a reboot.